cxx-sensors/src/main/java/org/sonar/cxx/sensors/utils/CxxIssuesReportSensor.java

```diff
@@ -192,12 +193,28 @@ private void saveViolation(SensorContext sensorContext, CxxReportIssue issue) {
       }
     }
 
+    for (CxxReportLocation location : issue.getFlow()) {
+      NewIssueLocation newIssueLocation = createNewIssueLocationFile(sensorContext, newIssue, location, affectedFiles);
+      if (newIssueLocation != null) {
+        newIssueFlow.add(newIssueLocation);
+      } else {
+        String msg = String.format("Failed to create new issue location from flow location %s", location);
+        LOG.debug(msg);

```

Comment:
@romanek-adam in case you log debug you don't need the validateRecovery. In case you wanna create an error it should be followed by validateRecovery. Still not sure what's the right way: Question is if such an "error" is so critical that you wanna stop analysis? Is only flow missing or is report wrong / displaying wrong data?
```Java
LOG.error(msg);
CxxUtils.validateRecovery(msg, getLanguage());
```
or
```Java
LOG.debug(msg);
```

Reply:
I assume this issue is not critical, I wouldn't stop the analysis. I'd rather clear the flow as in my initial commit, as to avoid displaying incomplete data.

```
LOG.debug("Failed to create new issue location from flow location {}", location);
newIssueFlow.clear();
break;
```

Is that OK?

Reply:
Yes, looks good.